File: test/randompathfortest/pipelines/test1/code/test1/graph/Reformat_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from test1.config.ConfigStore import *
from test1.udfs.UDFs import *

def Reformat_1(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0

File: perftest/e2epulloriginlag1/1721075067059/pipelines/Pipeline-Automation-PULL_ORIGIN1-1721075029606/code/src/main/scala/io/prophecy/pipelines/pipelineautomationpull_origin11721075029606/graph/Reformat_1.scala
package io.prophecy.pipelines.pipelineautomationpull_origin11721075029606.graph

import io.prophecy.libs._
import io.prophecy.pipelines.pipelineautomationpull_origin11721075029606.functions.PipelineInitCode._
import io.prophecy.pipelines.pipelineautomationpull_origin11721075029606.functions.UDFs._
import io.prophecy.pipelines.pipelineautomationpull_origin11721075029606.config.Context
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._
object Reformat_1 { def apply(context: Context, in: DataFrame): DataFrame = in }
